blosc/blosc.c

```diff
@@ -2136,7 +2165,7 @@ void blosc_destroy(void)
   g_initlib = 0;
   blosc_release_threadpool(g_global_context);
   my_free(g_global_context);
-  pthread_mutex_destroy(&global_comp_mutex);
+  pthread_mutex_destroy(global_comp_mutex);
```

Comment:
Should we require a `free(global_comp_mutex)` here?

Reply:
Also, I do not get why we need to create the `global_comp_mutex` in the heap, but I suppose you have good reasons for this.